ui/assets/js/common.js

```diff
@@ -13,7 +13,8 @@ var lib = {
     UrlToRepo: function(repo, path, line, rev) {
         var url = repo.url.replace(/\.git$/, ''),
             pattern = repo['url-pattern'],
-            anchor = line ? lib.ExpandVars(pattern.anchor, { line : line }) : '';
+            filename = path.replace(/^.*\//, ''),
```

Comment:
I agree that the suggestion clarifies the intention. I will push an update today.